docs/data-sources/lb.md

```diff
@@ -25,21 +25,19 @@ data "scaleway_lb" "by_id" {
 ## Argument Reference
 
 - `name` - (Optional) The IP address.
-  Only one of `name` and `lb_id` should be specified.
-
-- `lb_id` - (Optional) The ID.
-  Only one of `ip_address` and `lb_id` should be specified.
 
-- `zone` - (Optional) (Defaults to [provider](../index.md#zone) `region`) The [region](../guides/regions_and_zones.md#zones) in which the LB exists.
-
-- `project_id` - (Defaults to [provider](../index.md#project_id) `project_id`) The ID of the project the LB is associated with.
```

Comment:
Aren't zone and project_id configurable argument ? They should be added back in documentation

Reply:
They were removed as parameters but they should be added as attributes reference